client/state/reader/site-blocks/selectors/get-blocked-sites.js

```diff
@@ -2,7 +2,7 @@
  * External dependencies
  */
 import { map, pickBy } from 'lodash';
-import createSelector from 'calypso/lib/create-selector';
+import { createSelector } from '@automattic/state-utils';
```

Comment:
I'm actually really unsure where it should go, lol! I did an in-place find and replace to change these so that's why they're in various locations... they were already like that.

Reply:
Internal IMHO. To me, anything that's inside the repo is internal. Anything outside the repo is external. I believe it's as simple as that.